tests/test_apis/test_model_inference.py

```diff
@@ -18,6 +19,7 @@ def build_model(config_file):
     return model
 
 
+@pytest.mark.skipif(platform.system == 'Windows', reason='skip on windows')
```

Comment:
```suggestion
@pytest.mark.skipif(platform.system() == 'Windows', reason='skip on windows')
```
It should have been verified locally by running it in a way like `pytest -k test_model_inference`